Automattic/sensei
# Add/learner management edit course start date

This is useful for changing when a course is started

Added jquery-ui.css as this uses datepicker

To test

* Visit Sensei > Learner Management
* Select a course.
* You should see a new action, "edit start date" and an input to select
the new date.
* Date format accepted is like `2017-07-25`

![screen shot 2017-07-07 at 16 18 13](https://user-images.githubusercontent.com/500744/27959391-e98c6c42-632f-11e7-9786-7042dcb11e56.png)
